### DISCUSSION TITLE
Allowing to close #osu channel

### DISCUSSION THREAD

**Discussion by BlockHead007** [ppy/osu#25955] [Q&A] [answered]
The thing is the only channel I do use is #russian and I have highlights set up (in stable), so sometimes i do receive them from #osu channel because they're English.  So, I have to close #osu each time I am opening up stable because it opens every time I open the game.  Same thing is in lazer, but here it reopens each time someone sends message at the channel. Will there ever be ability to close that channel?

From 0:28
https://github.com/ppy/osu/assets/25553854/15c6cb63-9859-4273-a90b-d91912150b57

Lazer version 2023.1220.0

**Comment by peppy** [comment, marked as answer]
Closing the channel works fine here.
